### PR TITLE
#88 Narrative Updates

### LINKED ISSUE #88: Preparation for STU2 Ballot
**Real Submitter**
@regelkraut 

**Resource(s)/Profiles incl. Version**
HL7 Austria FHIR Core IG R4

**Links**
[HL7 Austria FHIR Core IG R4](https://fhir.hl7.at/r4-core-main/index.html)

**Priority**
Major

**Is your feature request related to a problem? Please describe.**
Various narrative parts for the IG are outdated and need to be brought up-to-date in order to reflect the recent changes that were done with STU2.

**Describe the solution you'd like**
Bring the narrative part of the IG up-to-date.

**Describe alternatives you've considered**
None

**Additional context**
None

### PR DESCRIPTION
**Pull request by regelkraut**
updated the index.md file to be up-to-date

Closes #88

### CODE REVIEW

File: input/pagecontent/index.md [hunk 1]
```diff
@@ -5,17 +5,27 @@
 This implementation guide is provided to support the use of FHIR® in Austria.
 
 This guide is a working specification. We anticipate that it will be implemented and tested by FHIR® system producers whose feedback will help improve its content. With this standard for trial use, we are looking for feedback on whether the following goals have been met:
-- The guide provides guidance on the [HL7® AT Core Patient Profile](StructureDefinition-at-core-patient.html).
+- The HL7 Austria FHIR Core Implementation Guide (IG) covers administrative Resources, like Patient, Organization, Practitioner, etc. that can be used on their own with no specific functional requirements for usage or it can be used for other more functional FHIR IGs in Austria via dependencies. It contains only those Profiles and Extension where adaptions for Austria were necessary.
```
Comment (oliver-krauss) [inline input/pagecontent/index.md:8]: are necessary

File: input/pagecontent/index.md [hunk 2]
```diff
@@ -5,17 +5,27 @@
...
+- It contains guidance on the [HL7® AT Core Patient Profile](StructureDefinition-at-core-patient.html).
   - Along with the Patient profile the representation of the common structure of address information within Austrian information systems ([HL7® AT Core Address Profile](StructureDefinition-at-core-address.html)) and the extension to encode the religious confession of a patient (only confessions registered in Austria) ([Patient Religion](StructureDefinition-at-core-ext-patient-religion.html)) has been specified.
-- This guide provides a set of sample Code Systems and ValueSets based on FSH files. Note, that Code Systems and ValueSets will generally be made available in a separate IG based on [TerminoloGit](https://gitlab.com/elga-gmbh/termgit).
-
-This implementation guide contains only those profiles and extension where adaptions for Austria were necessary.
+- Furthermore the codes for gender were extended to allow the usage of all official genders in Austria.
+- It defines the requirements for Organization, Practitioner or PractitionerRole to carry Austrian specific information (e.g. identifiers).
```
Comment (oliver-krauss) [inline input/pagecontent/index.md:13]: to carry information specific to Austria

File: input/pagecontent/index.md [hunk 3]
```diff
@@ -5,17 +5,27 @@
...
+- This guide contains no Code Systems or Value Sets on its own. Note, that Code Systems and Value Sets that are used in the various Profiles or Extensions of this guide will generally be made available via a separate IG on the [Austrian Terminology Server](https://termgit.elga.gv.at/) based on [TerminoloGit](https://gitlab.com/elga-gmbh/termgit).
+  - It does however contain an Extension and a Profile for the Value Set Resource, that allows to add the OID of Code System that is used within the Value Set, not just the URL reference.
```
Comment (oliver-krauss) [inline input/pagecontent/index.md:15]: however is a filler word

**Review by oliver-krauss — CHANGES_REQUESTED** [review]

File: input/pagecontent/index.md (hunk 1 above)
Comment (regelkraut) [reply input/pagecontent/index.md]: proposed change applied

File: input/pagecontent/index.md (hunk 2 above)
Comment (regelkraut) [reply input/pagecontent/index.md]: proposed change applied

File: input/pagecontent/index.md (hunk 3 above)
Comment (regelkraut) [reply input/pagecontent/index.md]: proposed change applied

File: input/pagecontent/index.md [hunk 4]
```diff
@@ -5,17 +5,27 @@
 This implementation guide is provided to support the use of FHIR® in Austria.
 
 This guide is a working specification. We anticipate that it will be implemented and tested by FHIR® system producers whose feedback will help improve its content. With this standard for trial use, we are looking for feedback on whether the following goals have been met:
-- The guide provides guidance on the [HL7® AT Core Patient Profile](StructureDefinition-at-core-patient.html).
+- The HL7 Austria FHIR Core Implementation Guide (IG) covers administrative Resources, like Patient, Organization, Practitioner, etc. that can be used on their own with no specific functional requirements for usage or it can be used for other more functional FHIR IGs in Austria via dependencies. It contains only those Profiles and Extension where adaptions for Austria are necessary.
```
Comment (gabriel0316) [inline input/pagecontent/index.md:8]:
```suggestion
- The HL7® Austria FHIR® Core Implementation Guide (IG) covers administrative resources, like Patient, Organization, Practitioner, etc. that can be used on their own with no specific functional requirements for usage or they can be used for other more functional FHIR® IGs in Austria via dependencies. It contains only those Profiles and Extension where adaptions for Austria are necessary.

```

File: input/pagecontent/index.md [hunk 5]
```diff
@@ -5,17 +5,27 @@
 This implementation guide is provided to support the use of FHIR® in Austria.
 
 This guide is a working specification. We anticipate that it will be implemented and tested by FHIR® system producers whose feedback will help improve its content. With this standard for trial use, we are looking for feedback on whether the following goals have been met:
-- The guide provides guidance on the [HL7® AT Core Patient Profile](StructureDefinition-at-core-patient.html).
+- The HL7 Austria FHIR Core Implementation Guide (IG) covers administrative Resources, like Patient, Organization, Practitioner, etc. that can be used on their own with no specific functional requirements for usage or it can be used for other more functional FHIR IGs in Austria via dependencies. It contains only those Profiles and Extension where adaptions for Austria are necessary.
+- This guide defines the minimum constraints on the FHIR resources to create the Austrian Core Profiles.
```
Comment (gabriel0316) [inline input/pagecontent/index.md:9]:
```suggestion
- This guide defines the minimum constraints on the FHIR® resources to create the Austrian Core Profiles. 

```

File: input/pagecontent/index.md [hunk 6]
```diff
@@ -5,17 +5,27 @@
...
-- The guide provides guidance on the [HL7® AT Core Patient Profile](StructureDefinition-at-core-patient.html).
+- The HL7 Austria FHIR Core Implementation Guide (IG) covers administrative Resources, like Patient, Organization, Practitioner, etc. that can be used on their own with no specific functional requirements for usage or it can be used for other more functional FHIR IGs in Austria via dependencies. It contains only those Profiles and Extension where adaptions for Austria are necessary.
+- This guide defines the minimum constraints on the FHIR resources to create the Austrian Core Profiles. 
+- It contains guidance on the [HL7® AT Core Patient Profile](StructureDefinition-at-core-patient.html).
   - Along with the Patient profile the representation of the common structure of address information within Austrian information systems ([HL7® AT Core Address Profile](StructureDefinition-at-core-address.html)) and the extension to encode the religious confession of a patient (only confessions registered in Austria) ([Patient Religion](StructureDefinition-at-core-ext-patient-religion.html)) has been specified.
```
Comment (gabriel0316) [inline input/pagecontent/index.md:11]:
```suggestion
  - Along with the Patient profile the representation of the common structure of address information within Austrian information systems ([HL7® AT Core Address Profile](StructureDefinition-at-core-address.html)) and the extension to encode the religious confession of a patient (only confessions registered in Austria) ([Patient Religion](StructureDefinition-at-core-ext-patient-religion.html)) have been specified.

```

File: input/pagecontent/index.md [hunk 7]
```diff
@@ -5,17 +5,27 @@
...
-- This guide provides a set of sample Code Systems and ValueSets based on FSH files. Note, that Code Systems and ValueSets will generally be made available in a separate IG based on [TerminoloGit](https://gitlab.com/elga-gmbh/termgit).
-
-This implementation guide contains only those profiles and extension where adaptions for Austria were necessary.
+- Furthermore the codes for gender were extended to allow the usage of all official genders in Austria.
+- It defines the requirements for Organization, Practitioner or PractitionerRole to carry information specific to Austria (e.g. identifiers).
+- This guide contains no Code Systems or Value Sets on its own. Note, that Code Systems and Value Sets that are used in the various Profiles or Extensions of this guide will generally be made available via a separate IG on the [Austrian Terminology Server](https://termgit.elga.gv.at/) based on [TerminoloGit](https://gitlab.com/elga-gmbh/termgit).
```
Comment (gabriel0316) [inline input/pagecontent/index.md:14]: We at ELGA have the governance that as long as we generally refer to code systems and value sets we write them in lower case. Only if we refer to FHIR resources we write CodeSystem or ValueSet.

File: input/pagecontent/index.md [hunk 8]
```diff
@@ -5,17 +5,27 @@
...
+  - It does contain an Extension and a Profile for the Value Set Resource, that allows to add the OID of Code System that is used within the Value Set, not just the URL reference.
```
Comment (gabriel0316) [inline input/pagecontent/index.md:15]:
```suggestion
  - It does contain an Extension and a Profile for the ValueSet resource, that allows to add the OID of code systems that are referenced within the ValueSet, not just the URL reference.

```

File: input/pagecontent/index.md [hunk 9]
```diff
@@ -5,17 +5,27 @@
...
+  - This profile is already used by the Austrian Terminology Server.
 
 **Download**: You can download this implementation guide in [NPM format](https://confluence.hl7.org/display/FHIR/NPM+Package+Specification) from [here](package.tgz).
 
-### Relation to Austrian EHR (ELGA)
+### Relation to Austrian EHR System (ELGA)
+
+Currently, the Austrian EHR system (ELGA) is mainly based on HL7® CDA®. In order to enable the creation of FHIR® resources that have a similar data structure this Implementation Guide provides mappings to the CDA® based implementation guides (e.g. [Mappings for Allgemeiner Implementierungsleitfaden v3](StructureDefinition-at-core-patient-mappings.html#mappings-for-allgemeiner-implementierungsleitfaden-v3-https-wiki)).
+Furthermore ELGA is using the HL7 Austria FHIR Core IG as a basis for their FHIR® Implementation guides.
```
Comment (gabriel0316) [inline input/pagecontent/index.md:23]:
```suggestion
Furthermore ELGA is using the HL7 Austria FHIR® Core IG as a basis for their FHIR® Implementation Guides.

```

File: input/pagecontent/index.md [hunk 10]
```diff
@@ -5,17 +5,27 @@
...
+
+### Dependencies
+This guide is based on the <a href="{{site.data.fhir.path}}">FHIR R4</a> specification.  In addition, this guide also relies on a number of parent implementation guides:
```
Comment (gabriel0316) [inline input/pagecontent/index.md:26]:
```suggestion
This guide is based on the <a href="{{site.data.fhir.path}}">FHIR® R4</a> specification.  In addition, this guide also relies on a number of parent implementation guides:

```

File: input/pagecontent/index.md (hunk 7 above)
Comment (regelkraut) [reply input/pagecontent/index.md:17]: That is a good proposal which makes sense to take over. This section of the narrative will be updated accordingly.

**Review by gabriel0316 — APPROVED** [review]

**Review by oliver-krauss — APPROVED** [review]

**Pull request merged**